products/samsungmobile.md

```diff
@@ -1015,15 +1018,15 @@ releases:
 -   releaseCycle: "galaxy-a-quantum"
     releaseLabel: "Galaxy A Quantum"
     releaseDate: 2020-05-22
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
     link: https://doc.samsungmobile.com/sm-a716s/skc/doc.html
 
 -   releaseCycle: "galaxy-tab-s6-lite"
     releaseLabel: "Galaxy Tab S6 Lite"
     releaseDate: 2020-05-16
-    eoas: false
-    eol: false
+    eoas: true
+    eol: true
```

Comment:
```suggestion
    eoas: 2023-05-16
    eol: 2025-05-16
```

https://androidspotlight.com/software-update-policy-for-every-samsung-device/, still on https://security.samsungmobile.com/workScope.smsb